trollmoves/client.py

```diff
@@ -254,6 +265,36 @@ def stop(self):
             self.subscriber = None
 
 
+def _is_push_message(msg):
```

Comment:
These functions are not only checking, but also acting on the message. Hence I think the `is_...` formulation is misleading. Maybe `_process_push_message` would be more appropriate?

Reply:
Yeah, I was thinking the same, but didn't come up with a better name. I think the `_process` prefix is better in any case, so I'll use that.

Reply:
I went with `_handle_*`, then the `is_message_already_handled()` method matches the naming.